src/events/templates/events/event_page.html

```diff
@@ -6,41 +6,88 @@
+            </div>
+            {% if is_in_person or is_hybrid %}
+                <div>
+                    <p>Location</p>
+                    <h4>{{ page.location|default_if_none:"TBC" }}</h4>
+                </div>
+                {% if page.location %}
+                    <div>
+                        <p>Room</p>
+                        <h4>
+                            {% if page.room %}
+                                {{ page.room }}
+                                {% if page.room_capacity %}with {{ page.room_capacity }} seats{% endif %}
+                            {% else %}
+                                TBC
+                            {% endif %}
  
-        {% if not page.is_past_event %}
-            {% if page.online_event_url and is_online %}
-                <a href="{{ page.online_event_url }}"
-                   class="dwds-button dwds-button--inline dwds-button--secondary">Join event online</a>
-            {% endif %}
-            {% if page.offline_event_url and is_in_person %}
-                <a href="{{ page.offline_event_url }}"
-                   class="dwds-button dwds-button--inline dwds-button--secondary">Register to join</a>
-            {% endif %}
-            {% if page.submit_questions_url %}
-                <a href="{{ page.submit_questions_url }}"
-                   class="dwds-button dwds-button--inline dwds-button--secondary">Submit a question</a>
+                        </h4>
+                    </div>
+                    {% if page.room %}
+                        <div>
+                            <p>Capacity</p>
+                            <h4>
+                                {% if page.room_capacity %}
+                                    {{ page.room_capacity }} seats
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </h4>
+                        </div>
+                    {% endif %}
+                {% endif %}
             {% endif %}
-        {% elif page.event_recording_url %}
- 
-            <a href="{{ page.event_recording_url }}"
-               class="dwds-button dwds-button--inline dwds-button--secondary">View event recording</a>
-        {% endif %}
-
-        <p>Audience: {{ page.get_audience_display }}</p>
-        <p>Location: {{ page.location|default_if_none:"-" }} {{ page.location.city|default_if_none:"" }}</p>
-        <p>Room: {{ page.room|default_if_none:"-" }} Capacity: {{ page.room_capacity|default_if_none:"-" }}</p>
+            <div>
+                <p>Audience</p>
+                <h4>{{ page.get_audience_display|default_if_none:"-" }}</h4>
+            </div>
+            <div>
+                <p>Event type</p>
+                {% if is_online or is_in_person %}
+                    <h4>{{ page.get_event_type_display }}</h4>
+                {% elif is_hybrid %}
```

Comment:
One to think about, I'm not too fussed either way:
this could be achieved with `elif is_online and is_in_person`

I can see pros/cons to both approaches so I'll let you make a call on which feels best

Reply:
I think i will use `else` instead, i don't know why i used `elif`:
```suggestion
               {% if is_online or is_in_person %}
                    <h4>{{ page.get_event_type_display }}</h4>
                {% else %}
```